src/scheduling.py

```diff
@@ -59,7 +60,7 @@ def login():
     # Mark: User Actions
 
     @app.route("/create_user", methods=["POST"])
-    def create_user():
+    def create_user():  # Creates user
```

Comment:
In the documentation were are starting to refer to users as employees. At some point we will update the code to refer to them as employees

Reply:
Ok I was unaware of that, will now change it to employees.

Reply:
Don't worry, there was no way for you to know.